charts/sqnc-node/templates/tests/checkPostDeployment.yaml

```diff
@@ -18,6 +19,47 @@ metadata:
     "helm.sh/hook": post-install
 spec:
   containers:
+    {{- if eq $.Values.node.role "validator" }}
```

Comment:
I think we should have a way of disabling this check. That way if a node is syncing lots we don't hit an issue here.